### PR TITLE
[i18n-KO] fix: docs: ko: sagemaker anchors and  `_toctree.yml`

### PR DESCRIPTION
**Pull request by jungnerd**
Co-authored-by: Wonhyeong Seo
<wonhseo@kakao.com>

# What does this PR do?
I fixed the anchors and `_toctree.yml`

Part of #20179 (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

Please review this PR: @sgugger

### CODE REVIEW

**Comment by wonhyeongseo** [comment]
We are getting a 500 error (probably due to spacing issues in `_toctree.yml`). Please do not merge until the problem is resolved.

**Review by sgugger — COMMENTED** [review]
I think the error in the doc preview comes from bad yaml syntax in the modified toctree.

**Comment by wonhyeongseo** [comment]
> I think the error in the doc preview comes from bad yaml syntax in the modified toctree.

Yes, I agree @sgugger . There were 2 section titles in one section. Editing it like so fixed the issue for me locally.
image

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -11,12 +11,13 @@
...
   title: 튜토리얼
 - sections:
-  - local: in_translation
-    title: (번역 중)
+    - local: sagemaker
+      title: Amazon SageMaker에서 학습 실행하기
+    - local: in_translation
+      title: (번역 중)
+  title: 일반 사용법
   title: How-to 가이드
```
Comment (wonhyeongseo) [inline docs/source/ko/_toctree.yml:19]: We should modify these lines to:
```yml
- sections:
  - sections:
    - local: sagemaker
      title: Amazon SageMaker에서 학습 실행하기
    - local: in_translation
      title: (번역 중)
    title: 일반 사용법
  - local: in_translation
    title: (번역 중)
  title: How-to 가이드
```
I understand the previous comment about how removing a single dash would resolve the issue might be misleading. Sorry for any confusion I may have caused.
Please let me know if you have further questions, @jungnerd !

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
We should modify L13-19 to the following:
```yml
- sections:
  - sections:
    - local: sagemaker
      title: Amazon SageMaker에서 학습 실행하기
    - local: in_translation
      title: (번역 중)
    title: 일반 사용법
  - local: in_translation
    title: (번역 중)
  title: How-to 가이드
```
I apologize for any confusion I may have caused with my suggestion that removing a single dash would resolve the issue. I understand it may have been misleading.
Please let me know if you have further questions, @jungnerd !

**Comment by wonhyeongseo** [comment]
Also when squashing the commits, please fix the typo on `Co-auth*e*red-by` to `Co-auth*o*red-by` and add arrow brackets (`<>`) to the email.
You may also use Github Desktop to ease the process. Thank you for your PR @jungnerd and feel free to ask me any questions.

**Comment by wonhyeongseo** [comment]
Great work, @jungnerd !
You solved the issue; now please squash the commits into one. You can use
- chat-gpt for the commit message and 
- Github Desktop to add co-authors
if you want. :raised_hands: Good night!

**Comment by wonhyeongseo** [comment]
@jungnerd we can remove the `_toctree.yml` change completely as we updated it in the upstream `ko: complete toctree` commit. After that and rebasing, this PR should be good to go! Let's try to do this on Thursday.

**Comment by jungnerd** [comment]
May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**